geonode/people/tests.py

```diff
@@ -911,3 +912,116 @@ def test_delete_a_user_with_resource(self):
+        for group in self.group_profiles:
+            self.assertTrue(bobby in group.get_managers())
+        # call the api that removes him from all assigned groups as manager
+        response = self.client.post(
+            path=f"{reverse('users-list')}/{bobby.pk}/remove_from_group_manager",
+            data={"groups": "ALL"},
+        )
+        self.assertTrue(response.status_code == 200)
+        # check that bobby is manager no more
+        for group in self.group_profiles:
+            self.assertFalse(bobby in group.get_managers())
+            self.assertTrue(group.title in response.json()["success"])
+
+    def test_remove_self_as_group_manager_list(self):
+        """
+        user is manager of some groups
+        and is quitting some of them
+        """
+        bobby = get_user_model().objects.get(username="bobby")
+
+        for group in self.group_profiles:
+            group.join(bobby)
+            group.promote(bobby)
+
+        self.assertTrue(self.client.login(username="bobby", password="bob"))
+        self.assertTrue(bobby.is_authenticated)
+        # assert that bobby is manager of these groups
+        for group in self.group_profiles:
+            self.assertTrue(bobby in group.get_managers())
+        # call the api that removes him from all assigned groups as manager
+        response = self.client.post(
+            path=f"{reverse('users-list')}/{bobby.pk}/remove_from_group_manager",
+            data={"groups": ",".join([str(group.group_id) for group in self.group_profiles[:3]])},
+        )
+        self.assertTrue(response.status_code == 200)
+        # check that bobby is  no more manager in the first groups
+        for group in self.group_profiles[:3]:
+            self.assertFalse(bobby in group.get_managers())
+            self.assertTrue(group.title in response.json()["success"])
+        # but still manager in the others
+        for group in self.group_profiles[4:]:
+            self.assertTrue(bobby in group.get_managers())
+
+    def test_remove_user_as_group_manager_empty(self):
+        """
+        user is manager of some groups
+        and is sending empty payload
+        """
+        bobby = get_user_model().objects.get(username="bobby")
+
+        for group in self.group_profiles:
+            group.join(bobby)
+            group.promote(bobby)
+
+        self.assertTrue(self.client.login(username="bobby", password="bob"))
+        self.assertTrue(bobby.is_authenticated)
+        # assert that bobby is manager of these groups
+        for group in self.group_profiles:
+            self.assertTrue(bobby in group.get_managers())
+        # call the api that removes him from all assigned groups as manager
+        response = self.client.post(
+            path=f"{reverse('users-list')}/{bobby.pk}/remove_from_group_manager",
+            data={"groups": ""},
+        )
+        self.assertTrue(response.status_code == 400)
+        self.assertTrue("Empty payload" in response.json()["error"])
```

Comment:
Can we give a more detailed error? This could be fine if no payload is passed at all. I guess here is better something like `No groups IDs were provided`